src/test/TestGsm4626Edge.t.sol

```diff
@@ -196,8 +196,8 @@ contract TestGsm4626Edge is TestGhoBase {
       address(USDC_4626_TOKEN),
       address(GHO_GSM_4626_FIXED_PRICE_STRATEGY)
     );
-    gsm.initialize(address(this), TREASURY, DEFAULT_GSM_USDC_EXPOSURE - 1);
-    GHO_TOKEN.addFacilitator(address(gsm), 'GSM Modified Exposure Cap', DEFAULT_CAPACITY);
+    gsm.initialize(address(this), TREASURY, DEFAULT_GSM_USDC_EXPOSURE - 1, address(GHO_RESERVE));
+    GHO_RESERVE.setWithdrawerCapacity(address(gsm), 100_000_000 ether);
```

Comment:
should we use `DEFAULT_CAPACITY` instead? It can be a good idea to rename or create new constant for `GHO_RESERVE_DEFAULT_LIMIT`